File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    answer = 0
    w = []
    h = []
    for i in dots:
        w.append(i[0])
        h.append(i[1])
    return abs(list(set(w))[0] - list(set(w))[1]) * abs(list(set(h))[0]-list(set(h))[1])

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
import java.util.Arrays;
class Solution {
    public int solution(int[][] dots) {
    int x1 = dots[0][0];
    int x2 = dots[0][0];
    int y1 = dots[0][1];
    int y2 = dots[0][1];

    for (int i = 1; i < dots.length; i++){
        if (dots[i][0] != x1){
            x2 = dots[i][0];
        }

        else if(dots[i][1] != y1){
            y2 = dots[i][1];
        }
    }
    return Math.abs(y2-y1) * Math.abs(x2-x1);
}
}